src/device-registry/utils/create-airqloud.js

```diff
@@ -127,6 +126,19 @@ const createAirqloud = {
       logObject("responseFromRegisterAirQloud", responseFromRegisterAirQloud);
 
       if (responseFromRegisterAirQloud.success === true) {
+        const payloads = [
```

Comment:
@Baalmart  

- we might not use `gcp-` at  the moment. Currently, topics are linked to clusters not cloud providers. e.g `strimzi.io/cluster: prod-kafka-cluster` in [one of the production files](https://github.com/airqo-platform/AirQo-api/blob/staging/k8s/topics/prod-airqo-streams-topics.yaml).

- Looks like you are creating a topic for each CRUD operation performed on an AirQlcoud. Could we use one topic and specify the operation performed on the AirQlcoud e.g something like
```json

{ 
  "action" : "create",
  "airqlcoud"  : {....airqlcoud details....} 
}
```

```json

{ 
  "action" : "update",
  "airqlcoud"  : {....airqlcoud details....} 
}
```


- Is `-0` the topic data version?? If so, how about using a schema registry to avoid creating multiple versions which might confuse the creators??

Reply:
Sound good, does this mean that the **action** is mapped to the topic in a separate configuration file? @NoahNsimbe

Reply:
The action is not mapped to anything. It just informs the consumer of what has happened to the site inorder for them to act accrodingly. For example if a consumer wants to send out alert notifications whenever a site is deleted, it just checks whether the action performed was `delete`, if not, it ignore the site. Yes, it would be good to have a topic for every action but actions performed on sites don't happen on a daily basis therefore messages will be less

Reply:
Okay, thanks for this clarification @NoahNsimbe 
How do we differentiate the actions of all use cases? That is: `Devices`, `Locations`, `Events,` `AirQlouds`, `Sites`.
In other words, how do we tell of the `delete` action is for either` Sites` or `AirQlouds`?

Reply:
by creating a topic for  every use case i.e `Devices`, `Locations`, `AirQlouds`, `Sites` etc.... but not creating a topic for every action in a use case. e.g 

**airqlcoud-topic**
```json
{ 
  "action" : "create",
  "airqlcoud"  : {....airqlcoud details....} 
}

```

**device-topic**
```json
{ 
  "action" : "delete",
  "device"  : {....device details....} 
}
```

**sites-topic**
```json
{ 
  "action" : "update",
  "site"  : {....site details....} 
}
```

Reply:
Yeah, awesome, sounds good @NoahNsimbe Thanks!